xml/System.Windows.Forms/Form.xml

```diff
@@ -1346,9 +1346,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Creates a new accessibility object for the <see cref="T:System.Windows.Forms.Form" /> control.</summary>
+        <returns>A new <see cref="T:System.Windows.Forms.AccessibleObject" /> for the control.</returns>
```

Comment:
Since the return type also shows the type next to it, natural language might look better:
```suggestion
        <returns>A new accessible object for the control.</returns>
```